sdk/purview/azure-analytics-purview-sharing/swagger/README.md

```diff
@@ -0,0 +1,16 @@
+## Generate autorest code
+
+```yaml
+input-file:
+- https://github.com/Azure/azure-rest-api-specs/blob/12cbd66750c4dfe302e9eca42b1fc727f8c02700/specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-02-15-preview/share.json
+output-folder: ../
+java: true
+regenerate-pom: false
+data-plane: true
+generate-tests: true
+artifact-id: azure-analytics-purview-sharing
+generate-samples: true
+namespace: com.azure.analytics.purview.sharing
```

Comment:
Regenerate using sync stack enabled:
enable-sync-stack: true
polling: {}

Reply:
Sync stack began to generate odd tests.  Discussed with @samvaity who will research for resolution at a later time.